lib/Runtime/Language/ValueType.h

```diff
@@ -354,6 +354,21 @@ struct DefaultComparer<ValueType> : public ValueTypeComparer
 {
 };
 
+__forceinline ValueType ValueType::Merge(const ValueType other) const
```

Comment:
In Linux build, CommonPal.h already defined __forceinline:
`#if defined(__GNUC__) || defined(__clang__)
#define __forceinline inline __attribute__((always_inline))
#else
#define __forceinline inline
#endif`

_ALWAYSINLINE are for template implementation only, because clang don't like `inline` on them